src/main/java/seedu/address/logic/commands/editcommands/EditStudentCommand.java

```diff
@@ -147,6 +154,16 @@ private String getDuplicatedTags(Set<Tag> currentTags, Set<Tag> newTags) {
         return duplicatedTags.toString();
     }
 
+    private boolean isDuplicatedEmail(List<Student> students, Email emailToEdit) {
+        for (Student student : students) {
+            Email curEmail = student.getEmail();
+            if (curEmail.equals(emailToEdit)) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
I'm not sure if it will make more sense to place this as a method in AddressBook instead